samples/sync_query.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import print_function  # For python 2/3 interoperability
+from samples.utils import get_service, query_paging, poll_query
+
+
+# [START sync_query]
+def sync_query(service, project_id, query, timeout=0):
+    """ Run a synchronous query"""
+    query_data = {
+                    'query': query,
+                    'timeoutMs': timeout,
+                 }
+
+    return service.jobs().query(
+            projectId=project_id,
+            body=query_data).execute()
```

Comment:
There is a separate function I wrote for polling jobs which checks if a job is `'DONE'` but `jobs.getQueryResults` API call returns a "query results" resource, not a job resource, which rather than having a `status.state` field, has a `jobComplete` field, and allows you to specify a timeout and retry number (evidently), while job polling does not.